FDBLibTLS/CMakeLists.txt

```diff
@@ -8,5 +8,5 @@ set(SRCS
   FDBLibTLSVerify.cpp
   FDBLibTLSVerify.h)
 
-add_library(FDBLibTLS ${SRCS})
+add_library(FDBLibTLS STATIC ${SRCS})
```

Comment:
I prefer removing the dependency cycle altogether, see #2145

Reply:
We decided that removing this dependency is out of scope for both #2145 and this PR. I filed #2201 to track that.